nightfall-optimist/src/services/database.mjs

```diff
@@ -218,8 +219,8 @@ export async function setRegisteredProposerAddress(address) {
   const connection = await mongo.connection(MONGO_URL);
   const db = connection.db(OPTIMIST_DB);
   logger.debug(`Saving proposer address ${address}`);
-  const data = { proposer: address };
-  return db.collection(METADATA_COLLECTION).insertOne(data);
+  const data = { _id: address };
```

Comment:
There are some conflicts in this file due to changes when adding a URL to the proposer. I thought I should update the url in the database as well (although I did not use this new info), but if you think it’s not needed, you can get rid of my changes.

Reply:
This seems like a good change though, so I will be sure to retain the URL.